ingest-app/src/cmr/ingest/services/jobs.clj

```diff
@@ -296,4 +307,8 @@
     ;; Run everyday at 12:20 pm. Chosen because it's least busy time for indexer historically and also
     ;; during business hours when people can debug issues. It's offset from the top of the hour so as
     ;; not to be at the same time as EDSC fetches all the collection metadata.
-    :daily-at-hour-and-minute [12 20]}])
+    :daily-at-hour-and-minute [12 20]}
+
+   {:job-type ReindexAutocompleteSuggestions
+    ;; Run everyday at 01:20. Chosen to be offset from the last job
+    :daily-at-hour-and-minute [01 20]}])
```

Comment:
The job scheduled run time should be `13:20` if it is going to be run 1 hour after the collection reindex job.